wikimedia/wikipedia-ios
# Update carthage files, including our custom NYTPhotoViewer

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T249274

### Notes
* The biggest concern here is the NYTPhotoViewer update. We've brought our custom one years of updates, and maintained our special display of the navbar. (I had to update some of our code to work w/ updated NYTPhotoViewer method signatures.)
* I checked for memory leaks, doesn't seem to have any.
 
Order of operations for this review/merge:
1. This PR should be checked
2. Merge in https://github.com/wikimedia/NYTPhotoViewer/pull/2/
3. Update the Cartfile in this branch to point to the proper NYTPhotoViewer branch
4. Check this PR again, merge it

### Test Steps
1. Ensure Picture of the Day works. 
2. Ensure images in article galleries work, including scrolling between them.
3. Nothing should be different than how the app currently operates.
